tensorboard/webapp/app_routing/effects/app_routing_effects.ts

```diff
@@ -159,6 +160,7 @@ export class AppRoutingEffects {
         options: {
           replaceState: navigationWithAbsolutePath.replaceState,
           browserInitiated: navigationWithAbsolutePath.browserInitiated,
+          resetNamespacedState: navigationWithAbsolutePath.resetNamespacedState,
```

Comment:
Given onPopState is not updated yet, would this value be null?

Reply:
Actually this handles merge(onNavigationRequested$, onInit$ and onPopState()) so it might not be null because of onNavigationRequested$.